wasm/libs/default_executor/src/lib.rs

```diff
@@ -38,6 +46,33 @@ impl Executor {
     }
 }
 
+fn visit_custom(executor: &mut Executor, isi: &CustomInstruction) {
```

Comment:
```suggestion
fn visit_custom_instructions(executor: &mut Executor, isi: &CustomInstruction) {
```

I hope we can rename this?

Reply:
but also, we should move this in the `iroha_executor/src/default.rs`. That also means that `iroha_multisig_data_model` should be merged into `iroha_executor_data_model`

Reply:
either that or we should move `default` permissions out of `iroha_executor_data_model` into a separate crate

Reply:
> rename this?

My understanding from [docs](https://github.com/hyperledger-iroha/iroha/blob/70e5aacf3290f4b143018bc6f21ce5dcecf38d18/crates/iroha_executor_derive/src/lib.rs#L78-L82
) was that the function should override with the same name as the default method. Or is your suggestion to rename overall, not specific to this `visit_custom` ?

Regarding moving to other crates, I think I can deepen my thoughts while reviewing your #5199

Reply:
> My understanding from [docs](https://github.com/hyperledger-iroha/iroha/blob/70e5aacf3290f4b143018bc6f21ce5dcecf38d18/crates/iroha_executor_derive/src/lib.rs#L78-L82) was that the function should override with the same name as the default method. Or is your suggestion to rename overall, not specific to this visit_custom ?

yeah, I suggest to rename the default method as well. `visit_custom` is ambiguous